app/src/containers/Map.js

```diff
@@ -28,7 +28,8 @@ const mapStateToProps = (state) => ({
   activeBasemap: state.map.activeBasemap,
   layerModal: state.map.layerModal,
   supportModal: state.map.supportModal,
-  layerLibraryModal: state.map.layerLibraryModal.open
+  layerLibraryModal: state.map.layerLibraryModal.open,
+  userPermissions: state.user.acl
```

Comment:
Why calling this `userPermissions` in containers/components, and `acl` in the app state?